src/api/general-services/pipeline-status.js

```diff
@@ -11,6 +11,48 @@ const privateSteps = [
   'DeleteCompletedGem2SWorker', 'LaunchNewGem2SWorker',
 ];
 
+// error code returned when an state machine execution arn does not exist (happens when
+// pulling or moving experiments across environments)
+const EXECUTION_DOES_NOT_EXIST = 'ExecutionDoesNotExist';
```

Comment:
perhas we could call it `MOCK_EXECUTION_DOES_NOT_EXIST` or `DEV_EXECUTION_DOES_NOT_EXIST` to make it clearer when it is (and should be) used. And do you think it would fit in constants.js?

Reply:
I moved it to the constants file. However, I did not rename it because it is a "real" error. We rely on it when mocking backend status but the error itself is not a mock and in some cases it could just happen normally (if somehow the state machines description is lost or the ARN changed) and naming it "mock" could create confusion.